hack/unit-test.sh

```diff
@@ -29,5 +29,14 @@ make test -C coredns
 echo "****** Testing Dashboard Operator ******"
 make test -C dashboard
 
+echo "****** Testing Flannel Operator ******"
+make test -C flannel
+
+echo "****** Testing Kube-Proxy Operator ******"
+make test -C kubeproxy
```

Comment:
So looking at the failure, I think it's picking up the KUBERNETES_SERVICE_HOST from prow!

I think we can just unset KUBERNETES_SERVICE_HOST before calling this e.g. `KUBERNETES_SERVICE_HOST= make test -C kubeproxy`

(Or just `unset KUBERNETES_SERVICE_HOST` at the top of hack/unit-test.sh)

Reply:
Aah yes, I'll add that!